syncback/Tiltfile

```diff
@@ -22,6 +24,8 @@ def syncback(name, k8s_object, src_dir, target_dir='.', paths=None, container=''
     :param verbose (bool, optional): if true, print additional rsync information.
     :return:
     """
+    # if no local rsync/insufficient version, bail
+    local('{} {}'.format(verify_rsync_path, MIN_LOCAL_RSYNC_VERSION))
```

Comment:
might make more sense to either do this on extension load, or in the local_resource itself? doing it in the syncback() function seems like the worst of all worlds